src/clover/analysis.py

```diff
@@ -0,0 +1,2852 @@
+                        ColumnHeader.HW_TANK_TEMPERATURE.value
+                    ].values,
+                    (31, 24),
+                ),
+                axis=0,
+            )
+
+            # Plot the average collector output temperature
+            renewable_fraction_january = np.mean(
+                np.reshape(
+                    simulation_output[0 : 31 * 24][
+                        ColumnHeader.HW_RENEWABLES_FRACTION.value
+                    ].values,
+                    (31, 24),
+                ),
+                axis=0,
+            )
+            renewable_fraction_march = np.mean(
+                np.reshape(
+                    simulation_output[HOURS_UNTIL_MARCH : HOURS_UNTIL_MARCH + 31 * 24][
+                        ColumnHeader.HW_RENEWABLES_FRACTION.value
+                    ].values,
+                    (31, 24),
+                ),
+                axis=0,
+            )
+            renewable_fraction_may = np.mean(
+                np.reshape(
+                    simulation_output[HOURS_UNTIL_MAY : HOURS_UNTIL_MAY + 31 * 24][
+                        ColumnHeader.HW_RENEWABLES_FRACTION.value
+                    ].values,
+                    (31, 24),
+                ),
+                axis=0,
+            )
+            renewable_fraction_july = np.mean(
+                np.reshape(
+                    simulation_output[HOURS_UNTIL_JULY : HOURS_UNTIL_JULY + 31 * 24][
+                        ColumnHeader.HW_RENEWABLES_FRACTION.value
+                    ].values,
+                    (31, 24),
+                ),
+                axis=0,
+            )
+
+            ax1.plot(
+                collector_output_temperature_january,
+                label="january collector output temp.",
+            )
+            # ax1.plot(collector_output_temperature_march, label="march collector output temp.")
+            # ax1.plot(collector_output_temperature_may, label="may collector output temp.")
+            ax1.plot(
+                collector_output_temperature_july, label="july collector output temp."
+            )
+            ax1.plot(
+                hot_water_tank_temperature_january,
+                ":",
+                label="january tank temp.",
+                color="C0",
+            )
+            # ax1.plot(hot_water_tank_temperature_march, ":", label="march tank temp.", color="C2")
+            # ax1.plot(hot_water_tank_temperature_may, ":", label="may tank temp.", color="C3")
+            ax1.plot(
+                hot_water_tank_temperature_july,
+                ":",
+                label="july tank temp.",
+                color="C1",
+            )
+            ax1.legend(loc="upper left")
+
+            ax2 = ax1.twinx()
+            ax2.plot(
+                renewable_fraction_january, "--", label="january renewables fraction"
+            )
+            # ax2.plot(renewable_fraction_march, "--", label="march renewables fraction")
```

Comment:
Are these still relevant?

Apologies if you get a notification for every time I've added this comment